### ISSUE TITLE
1.7.0 Release - Mac OS build fails due to bus error

### ISSUE THREAD

**Issue by filthmancer** [Unity-Technologies/unity-gaming-services-cli#9]
Running the mac 1.7.0 release will return `bus error  ./ugs -h`. 
From the system logs:

```
CODE SIGNING: 14212[ugs] vm_map_enter: curprot cannot be write+execute. failing
CODE SIGNING: 14212[ugs] vm_map_protect:6281(0x0,0x0,0x7) can't have both write and exec at the same time
```

1.6.0 works fine, so must be a signing issue with the new release. This is on M3.

Cheers 🙏

**Comment by GabrielBelmonteUnity** [comment]
Hey!
The signing system changed, its been fixed and will be part of the 1.7.1 release.

**Comment by GabrielBelmonteUnity** [comment]
Please try out 1.7.1: https://github.com/Unity-Technologies/unity-gaming-services-cli/releases/
